src/modules/ModuleInsightsBento/ModuleInsightsBento.tsx

```diff
@@ -67,7 +67,7 @@ export const ModuleInsightsBento = ({
           </h4>
 
           <Button asChild>
-            <Link link={link} {...inspector("exploreInsightsButton")}>
+            <Link link={link} {...inspector("cta")}>
```

Comment:
Same thing here where you don't need the `readFragment` above or to pass children manually.